clinica/iotools/converters/oasis_to_bids/oasis_to_bids.py

```diff
@@ -118,31 +118,15 @@ def _create_sessions_tsv(
 
         write_sessions_tsv(bids_dir, sessions_dict)
 
-        return sessions_dict
-
     def _create_scans_tsv(
         self,
-        clinical_data_dir: Path,
         bids_dir: Path,
-        bids_ids: list[str],
-        sessions: dict,
-    ):
-        from clinica.iotools.bids_utils import (
-            StudyName,
-            create_scans_dict,
-            write_scans_tsv,
+    ) -> None:
+        from clinica.iotools.converters.oasis_to_bids.oasis_to_bids_utils import (
```

Comment:
Why not `from .oasis_to_bids_utils import write_scans` ?

Reply:
No real reason ! I will change